docs/book/src/cronjob-tutorial/testdata/emptymain.go

```diff
@@ -89,6 +90,40 @@ func main() {
 		os.Exit(1)
 	}
 
+	/*
+
+	Note that the Manager can restrict the namespace that all controllers will watch for resources by:
+
+	```
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Scheme: scheme,
+		Namespace: namespace,
+		MetricsBindAddress: metricsAddr}
+	)
+	```
+
+	The above example will change the scope of your project to a single Namespace. In this scenario,
+	it is also suggested to restrict the provided authorization to this namespace by replacing the default
+	ClusterRole and ClusterRoleBinding to Role and RoleBinding respectively
+	For further information see the kubernetes documentation about Using RBAC Authorization:
+	https://kubernetes.io/docs/reference/access-authn-authz/rbac/.
+
+	Also, it is possible to use the MultiNamespacedCacheBuilder to watch a specific set of namespaces;
+
+	```
```

Comment:
Same here

Reply:
```
	```go
	var namespaces []string // List of Namespaces
	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
	    Scheme: scheme,
	    NewCache: cache.MultiNamespacedCacheBuilder(namespaces),
	    MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
	})
	```
```